src/building/industry.c

```diff
@@ -69,6 +91,64 @@ int building_is_workshop(building_type type)
     return type >= BUILDING_WINE_WORKSHOP && type <= BUILDING_POTTERY_WORKSHOP;
 }
 
+static int building_produces(building_type type)
```

Comment:
I don't like this static list.

Much better to cycle through the resource info list and return the reosurce if `resource->industry` matches the building, or return `RESOURCE_NONE`if it doesn't match. That will transparently handle any future resources we add.

Also, this function should also be moved to `game/resource.c` and be called something like `resource_produced_by_building_type`